web/src/components/FilingsCalendarTaxAccess.test.tsx

```diff
@@ -337,16 +337,34 @@ describe("<FilingsCalendarTaxAccess />", () => {
         return expect(currentUserData().taxFilingData.state).toEqual("FAILED");
       });
       await screen.findByRole("alert");
-      expect(screen.getByRole("alert")).toHaveTextContent(
-        markdownToText(Config.taxCalendar.failedErrorMessageMarkdown)
-      );
+      expect(screen.getByRole("alert")).toHaveTextContent(Config.taxCalendar.failedErrorMessageHeader);
+      expect(screen.getByRole("alert")).toHaveTextContent(Config.taxCalendar.modalBusinessFieldErrorName);
       expect(screen.getByText(Config.taxCalendar.failedTaxIdHelper)).toBeInTheDocument();
       expect(screen.getByText(Config.taxCalendar.failedBusinessFieldHelper)).toBeInTheDocument();
       expect(
         screen.queryByText(Config.taxCalendar.failedResponsibleOwnerFieldHelper)
       ).not.toBeInTheDocument();
     });
 
+    it("states business name specific field error in alert if api fails", async () => {
```

Comment:
I see in your code that you have logic that changes the error message shown depending on whether it needs to show responsible owner name or business name. That part isn't tested for the Responsible Owner Name because this test only exists for the Public Filing case. You should have a corresponding test to cover the rest of the logic you coded.

Reply:
yeah that makes sense, I forgot to replicate the test for the parallel scenario. I will fix that.